ci/pipeline.yml

```diff
@@ -160,13 +105,6 @@ jobs:
         passed: [set-pipeline]
         trigger: true
         params: {depth: 1}
-      - put: src
```

Comment:
Are these no longer needed?

Reply:
Since we removed the `cogito` resource type, we can no longer send status updates to branches and just to PR's. We only use PR status updates now but it may be worth investigating options to send status to a branch.

Reply:
Thanks; I think I need to address this properly in a few places.